common/interfaces/desktop_app_interface/usb_event.c

```diff
@@ -62,7 +62,9 @@
  *****************************************************************************/
 #include <string.h>
```

Comment:
Please move this file in core/ directory

Reply:
will relocate this file in further PR.